### PR TITLE
Adding a feature-animation example

### PR DESCRIPTION
**Pull request by pgiraud**
Here's a new example showing how one can animate the rendering of features on the map using postcompose and vectorContext.
It's closely related to this question on StackOverflow. http://stackoverflow.com/questions/23857161/how-to-animate-feature-in-openlayers3

Please review.

### CODE REVIEW

File: examples/feature-animation.js [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+var duration = 3000;
+function flash(feature) {
+  var start = new Date().getTime();
+  var listenerKey;
+
+  function animate(event) {
+    var vectorContext = event.vectorContext;
+    var frameState = event.frameState;
+    var flashGeom = feature.getGeometry().clone();
+    var ellapsed = frameState.time - start;
```
Comment (bartvde) [inline examples/feature-animation.js]: elapsed instead of elapsed

File: examples/feature-animation.js [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+    var ellapsedRatio = ellapsed / duration;
```
Comment (bartvde) [inline examples/feature-animation.js]: ellapseRatio

**Comment by bartvde** [comment]
LGTM

**Comment by elemoine** [comment]
LGTM too!

**Pull request merged**